publish-pipeline/async-questionset-publish/src/main/resources/async-questionset-publish.conf

```diff
@@ -36,4 +36,8 @@ redis {
   database {
     qsCache.id = 0
   }
-}
\ No newline at end of file
+}
+cloudstorage.metadata.replace_absolute_path=true
```

Comment:
@krgauraw instead of hardcoding can we take these values from env

Reply:
@pallakartheekreddy this is local config